spec/support/rails_template.rb

```diff
@@ -64,7 +105,12 @@ def display_name
 
 # Generate a model with string ids
 generate :model, "tag name:string"
-gsub_file(Dir['db/migrate/*_create_tags.rb'][0], /\:tags\sdo\s.*/, ":tags, id: false, primary_key: :id do |t|\n\t\t\tt.string :id\n")
+gsub_file(Dir['db/migrate/*_create_tags.rb'][0],
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.